src/components/VideoPlayer/index.js

```diff
@@ -52,11 +54,16 @@ export default class VideoPlayer extends React.PureComponent {
 
   componentDidMount () {
     if (window.bc && window.videojs) {
-      window.bc(this.playerRef.current, {
-        playbackRates: [0.5, 1, 1.5, 2],
-      })
-      this.video = window.videojs(this.playerRef.current)
-      this.video.ready(this.handlePlayerReady)
+      this.setupVideo()
+    } else {
+      const { playerId, accountId } = this.props
+
+      const bcScript = document.createElement('script')
```

Comment:
Maybe we can wait for this [PR](https://github.com/yankaindustries/mc-components/pull/50) and reuse [this](https://github.com/yankaindustries/mc-components/blob/ba49852937d02ef53520e1ea4d7f6a295ec44aaa/src/utils/brightcove.js#L5) `brightcove.js` function.